test/test_callback.py

```diff
@@ -0,0 +1,31 @@
+import threading
+
+from absl.testing import absltest
+import torch
+import torch_xla
+from torch_xla.experimental import callback
+
+
+class TestExperimentalCallback(absltest.TestCase):
+
+  @staticmethod
+  @torch_xla.compile
+  def executable():
+    a, b = torch.randn((100, 100), device=torch_xla.device()), torch.randn(
+        (100, 100), device=torch_xla.device())
+    return a @ b
+
+  def test_callback(self):
+    event = threading.Event()
+    c = self.executable()
+
+    def cb(tensor):
```

Comment:
is it difficult to support the cases where `call_back` takes more than just the tensor value? `call_back` might be a general python function that `tensor` is just one of the inputs.

Reply:
You should `functools.partial` in that case to add other arguments to the callable https://docs.python.org/3/library/functools.html#functools.partial